packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -222,3 +222,23 @@ def assert_packages_match_active_buildinfo(package_ids):
 
         assert set(active_package_ids) <= set(package_ids)
         assert_packages_match_active_buildinfo(active_package_ids)
+
+
+def test_packaging_api(dcos_api_session):
+    """Test the Cosmos API (/package) wrapper
+    """
+    install_response = dcos_api_session.cosmos.install_package('kafka', '1.1.9-0.10.0.0')
```

Comment:
The most recent version of [kakfa](https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/K/kafka/39/config.json#L97) says it needs 3 brokers, and each broker needs 1.0 cpus with 2304 mem, 2048 heap, and 5000 disk.